src/vs/workbench/contrib/chat/browser/chatEditing/chatEditingSession.ts

```diff
@@ -708,13 +708,20 @@ export class ChatEditingSession extends Disposable implements IChatEditingSessio
 		let didComplete = false;
 
 		return {
-			pushText: edits => {
+			pushText: (edits) => {
```

Comment:
@connor4312 @DonJayamanne Can we align these and just have `pushEdits(uri, edits)` because all these call `_acceptEdits`

Reply:
Sorry I missed this, will take a look.